website/app-templates/smarty/help-pages/en/help.html

```diff
@@ -579,50 +535,55 @@ <h3>for <a href="https://www.mozilla.com/firefox/">Mozilla FireFox</a></h3>
             <p>
-                You also need an account on geokrety.org. After installing GKT
-                in your browser, you will see an extra
-                [<strong> Geo<span style="color: white; background-color: #303030;">Krety</span>.org </strong>]
-                panel on the right hand side of every GC cache page.
-                From the list of GeoKrety pick the one you wish to drop and
-                click "Drop selected GK" - you will be redirected to the standard
-                logging page with both the GC Code and coordinates already filled
+                Then click "Log any GeoKret here" and you will be you will be redirected to the standard
+                GeoKrety logging page with both the GC Code and coordinates already filled
                 in. Check the date, add a comment and you're done.
             </p>
 
 
-            <h3>Here is how GKT looks like</h3>
+            <h4>Here is what GKT looks like</h3>
             <p>
-                <img src="{GK_CDN_IMAGES_URL}/help/gkt_screenshot030_1.png" alt="GKT screenshot">
-                <img src="{GK_CDN_IMAGES_URL}/help/gkt_screenshot030_2.png" alt="GKT screenshot">
-                <img src="{GK_CDN_IMAGES_URL}/help/gkt_screenshot030_3.png" alt="GKT screenshot">
+                <img src="{GK_CDN_IMAGES_URL}/help/GeoKrety_inventory_on_cache_page.png" alt="GKT screenshot">
             </p>
         </li>
 
         <li class="list-group-item">
             <a class="anchor" id="manual-log"></a>
-            <h4>Manually, using copy and paste</h4>
+            <h3>2. Manually, using copy and paste</h4>
             <p>
-                Go to <a href="{'move_create'|alias}">Log a GeoKret</a> page
+                Go to the <a href="{'move_create'|alias}">Log a GeoKret</a> page.
+                Fill in the Tracking Code of the GeoKret you wish to log,
+                
                 and under the section <strong>New location</strong> enter the
                 following:
 
             </p>
             <ol>
-                <li>in the waypoint field, enter the GC Code of the cache (eg. GC1AQ2N).</li>
-                <li>in the lat/lon field, enter the coordinates of the cache (you can copy-paste it directly from cache page or <a href="#acceptableformats">use other acceptable formats</a>)</li>
+                <li>In the waypoint field, enter the GC Code of the cache (eg. GC2586K).</li>
+                <li>Optionally, change the Coordinates if they are different from the cache listing. 
+                    See <a href="#acceptableformats">here</a> for acceptable formats.</li>
             </ol>
             <p>Here is an example:</p>
-            <p><img src="{GK_CDN_IMAGES_URL}/help/location-dla-gc3.png" alt="how to log GK for GC caches" class="center-block img-responsive" /></p>
+            <p><img src="{GK_CDN_IMAGES_URL}/help/Manual_copy_paste.png" alt="how to log GK for GC caches" class="center-block img-responsive" /></p>
```

Comment:
The constant GK_CDN_IMAGES_URL refers to another repo. Images should be committed there. I think the link is given in the related issue. Will check after the review

Reply:
Images go there https://github.com/geokrety/GeoKrety-Static/tree/main/content/images/help